tests/test_cli.py

```diff
@@ -96,23 +98,27 @@ def test_deposit_success(
         uploaded_metadata = s3_class.client.get_object(
             Bucket="awd", Key="10.1002-term.3131.json"
         )
-        assert uploaded_metadata["ResponseMetadata"]["HTTPStatusCode"] == 200
+        assert uploaded_metadata["ResponseMetadata"]["HTTPStatusCode"] == STATUS_CODE_200
```

Comment:
This was new to me, but we might consider using enumerated values from the `http.HTTPStatus` stdlib:

```python
from http import HTTPStatus

assert uploaded_metadata["ResponseMetadata"]["HTTPStatusCode"] == HTTPStatus.OK
```

The constant `STATUS_CODE_200` might be better than just the integer `200`, but it's still kind of circular in the sense it's still just a `200`, where `HTTPStatus.OK` is semantically meaningful as the request was good / successful.

Disclaimer: I've never used this, but I like it, and might be worth considering here.

Reply:
This is a great suggestion, thanks so much!